uo_sys_framework/AILinkModule/cfgXEH/init/PreInit.sqf

```diff
@@ -0,0 +1,25 @@
+#define COMPONENT AILink
+#include "\x\UO_FW\addons\Main\script_macros.hpp"
+UO_FW_EXEC_CHECK(ALL);
+
+["UO_FW_SettingsLoaded",
+{
+	if (!(UO_FW_GETMVAR(AILink_Enabled,false))) exitWith {};
+	["UO_FW_RegisterModuleEvent", ["AI link", "Shares targeting information between AI groups based on radios.", "TinfoilHate and Sacher"]] call CBA_fnc_globalEvent;
+	tin_AILink_startDelay 	= (UO_FW_GETMVAR(AILink_StartDelay,30));		//Delay in seconds when starting the mission until the first loop runs.
```

Comment:
I think we can just use the mission vars themselves in the main function instead of defining global var copies, this was only for player defined settings.sqf